tools/elf2efi.py

```diff
@@ -231,6 +231,7 @@ def use_section(elf_s: ELFSection) -> bool:
         ".note.gnu.build-id",
         ".rel.dyn",
         ".rela.dyn",
+        ".relr.dyn",
```

Comment:
Heh, I should've actually looked at the file, and then I could've given a PR myself. Thanks!

Reply:
can you confirm this fixes the issue for you?

Reply:
Yeah, it fixes it for me.